### ISSUE TITLE
Hello Again.

### ISSUE THREAD

**Issue by NycuRO** [cuberite/cuberite#4191]
On April 20, I suggested such an idea, but it was not really taken into account because there was no time to deal with it. So I thought, after a few good months, to come back with this idea. I'll probably be refused again, but if you do not risk, you do not win.

The idea is to create the "Cuberite-BE" software for Pocket Edition aka Bedrock Edition.

So, what do you say? : D

**Comment by madmaxoft** [comment]
Hello again. Sorry, but this is highly unlikely to happen any time soon (which means a few years at least). There's very few people actually working on Cuberite right now, and there are features that are much more higher on the priority list.

**Comment by NycuRO** [comment]
So, no changes? :(

**Comment by beezTEM** [comment]
I personally know many people that would love to use this, even if it takes a while for a release, I would be willing to help test as I am also excited to see any progress on this.

**Comment by lukemagdalin** [comment]
RealJustADev They don't need help testing it they need help developing it as madmaxoft said "There's very few people actually working on Cuberite right now, and there are features that are much more higher on the priority list." meaning they are busy doing other things for cuberite.

**Comment by DarkoGNU** [comment]
I don't think that a separate project for Bedrock is a good idea. It's possible to join Java Edition servers using a Bedrock client with a proxy. There are at least two proxies:

- [DragonProxy](https://github.com/DragonetMC/DragonProxy) (abandoned)
- [Geyser](https://github.com/GeyserMC/Geyser) (actively developed)

Using Geyser will be impossible until 1.18.2 is supported in Cuberite - Geyser supports only the latest protocol.

I've tried using DragonProxy, but I'd have to downgrade Bedrock and I decided it's not worth my time

**Comment by Janmm14** [comment]
Idk if it works but one could maybe combine geyser on bungeecord with viaversion on bungeecord to make it work right now.

**Comment by DarkoGNU** [comment]
I'll try.

Also, using a proxy isn't necessarily the best solution for Cuberite. Just like it supports multiple Java protocol versions right now, it should also be possible to add support for the Bedrock protocol. It would be harder than just adding support for another version, though - Bedrock uses UDP and its protocol is fairly different.

So, just making sure that Cuberite is compatible with one of the proxies might turn out to easier and better - another open-source project would be taking care of Bedrock

**Comment by dxomg** [comment]
> Idk if it works but one could maybe combine geyser on bungeecord with viaversion on bungeecord to make it work right now.

I've already tried this way before this was posted
I can confirm this works but on older builds the bedrock client got lagged
